Cargo.toml

```diff
@@ -13,7 +13,7 @@ async-trait = "0.1.66"
 bitcoin = { version = "0.29.2", features = ["rand"] }
 futures = "0.3.26"
 home = "0.5.5"
-lightning = "0.0.116"
+lightning = "0.0.118"
```

Comment:
manually tested that our base functionality is unaffekted by this bump, all in order 👍